src/main/java/me/qoomon/maven/gitversioning/VersioningMojo.java

```diff
@@ -34,8 +34,8 @@
       threadSafe = true)
 public class VersioningMojo extends AbstractMojo {
 
+    public static final String GIT_VERSIONING_POM_NAME = ".git-versioned-pom.xml";
```

Comment:
I think you don't need to have a change here, package visibility should be enough.